test/logger.go

```diff
@@ -94,9 +94,10 @@ func collectManagementClusterLogs(bootstrapClusterProxy *e2e.AzureClusterProxy,
 	}
 
 	framework.DumpAllResources(context.TODO(), framework.DumpAllResourcesInput{
-		Lister:    bootstrapClusterProxy.GetClient(),
-		Namespace: *namespace,
-		LogPath:   workLoadClusterLogPath,
+		Lister:         bootstrapClusterProxy.GetClient(),
+		KubeConfigPath: bootstrapClusterProxy.GetKubeconfigPath(),
```

Comment:
```suggestion
		KubeConfigPath:       bootstrapClusterProxy.GetKubeconfigPath(),
		ClusterctlConfigPath: getClusterctlConfigPath(),
		Namespace:            *namespace,
		LogPath:              workLoadClusterLogPath,
	})
}

func getClusterctlConfigPath() string {
	config := os.Getenv("CLUSTERCTL_CONFIG")
	if config == "" {
		return path.Join(getArtifactsFolder(), "repository", "clusterctl-config.yaml")
	}

	return config
}
```

I see the problem--there's no obvious way to get the clusterctl config path from the e2e package. We could export the variable as `e2e.ClusterctlConfigPath`, or we could add a helper function like this.